r/R/rzeppelin/R/globals.R

```diff
@@ -0,0 +1,3 @@
+lEtTeRs <- c(letters,LETTERS)
```

Comment:
It's [BSD-3 clause](https://cran.r-project.org/web/packages/rscala/index.html), https://github.com/cran/rscala/blob/master/R/globals.R